docs/Troubleshooting.md

```diff
@@ -1,137 +1,125 @@
+   - How to check: run `kill -3 PID`. Healthy JVM process should print
+     a thread dump and heap info in its console.
+
+### Target JVM failed to load libasyncProfiler.so
+
+The connection with the target JVM has been established, but JVM is unable to load profiler shared library.
+Make sure the user of JVM process has permissions to access `libasyncProfiler.so` by exactly the same absolute path.
+For more information see [#78](https://github.com/async-profiler/async-profiler/issues/78).
+
+### Perf events unavailable
+
+`perf_event_open()` syscall has failed. Typical reasons include:
+
+1. `/proc/sys/kernel/perf_event_paranoid` is set to restricted mode (>=2).
+2. seccomp disables `perf_event_open` API in a container.
+3. OS runs under a hypervisor that does not virtualize performance counters.
+4. perf_event_open API is not supported on this system, e.g. WSL.</br>
+
+</br>For permissions-related reasons (such as 1 and 2), using `--fdtransfer` while running the profiler
+as a privileged user may solve the issue.
+
+If changing the configuration is not possible, you may fall back to
+`-e ctimer` profiling mode. It is similar to `cpu` mode, but does not
+require perf_events support. As a drawback, there will be no kernel
+stack traces.
+
+### No AllocTracer symbols found. Are JDK debug symbols installed?
+
+The OpenJDK debug symbols are required for allocation profiling for applications developed
+with JDK prior to 11. See [Installing Debug Symbols](ProfilingModes.md#installing-debug-symbols) for more
+details. If the error message persists after a successful installation of the debug symbols,
+it is possible that the JDK was upgraded when installing the debug symbols.
+In this case, profiling any Java process which had started prior to the installation
+will continue to display this message, since the process had loaded
+the older version of the JDK which lacked debug symbols.
+Restarting the affected Java processes should resolve the issue.
+
+### VMStructs unavailable. Unsupported JVM?
+
+JVM shared library does not export `gHotSpotVMStructs*` symbols -
+apparently this is not a HotSpot JVM. Sometimes the same message
+can be also caused by an incorrectly built JDK
+(see [#218](https://github.com/async-profiler/async-profiler/issues/218)).
+In these cases installing JDK debug symbols may solve the problem.
+
+### Could not parse symbols from <libname.so>
+
+Async-profiler was unable to parse non-Java function names because of
+the corrupted contents in `/proc/[pid]/maps`. The problem is known to
+occur in a container when running Ubuntu with Linux kernel 5.x.
+This is the OS bug, see <https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1843018>.
+
+### Could not open output file
+
+Output file is written by the target JVM process, not by the profiler script.
+Make sure the path specified in `-f` option is correct and is accessible by the JVM.
+
+- No Java stacks will be collected if `-XX:MaxJavaStackTraceDepth` is zero
```

Comment:
A title is needed, otherwise it reads as continuation of `Could not open output file` section.

"Limitations" maybe?